static/js/components/SurveyLinkList.jsx

```diff
@@ -26,83 +26,85 @@ const SurveyLinkList = ({ ra, dec, id }) => {
   return (
     <div className={styles.SurveyLinkList}>
       <SurveyLink
-        name="Gaia DR2"
-        url={`http://vizier.u-strasbg.fr/viz-bin/VizieR?-source=I/345/gaia2&-out.add=_r&-out.add=2C_DEJ&-sort=_r&-to=&-out.max=20&-meta.ucd=2&-meta.foot=1&-c=${ra_hrs}+${dec_hrs}&-c.rs=10`}
-      />
-      <SurveyLink
-        name="NED"
-        url={`http://nedwww.ipac.caltech.edu/cgi-bin/nph-objsearch?lon=${ra}d&lat=${dec}d&radius=1.0&search_type=Near+Position+Search`}
+        name="ADS"
+        url={`http://adsabs.harvard.edu/cgi-bin/nph-abs_connect?db_key=AST&db_key=PRE&qform=AST&arxiv_sel=astro-ph&arxiv_sel=cond-mat&arxiv_sel=cs&arxiv_sel=gr-qc&arxiv_sel=hep-ex&arxiv_sel=hep-lat&arxiv_sel=hep-ph&arxiv_sel=hep-th&arxiv_sel=math&arxiv_sel=math-ph&arxiv_sel=nlin&arxiv_sel=nucl-ex&arxiv_sel=nucl-th&arxiv_sel=physics&arxiv_sel=quant-ph&arxiv_sel=q-bio&sim_query=YES&ned_query=YES&adsobj_query=YES&aut_logic=OR&obj_logic=OR&author=&object=iptf${id}%%0D%%0Aptf${id}%%0D%%0A${ra}+${
+          dec > 0 ? "%2B" : ""
+        }${dec}&start_mon=&start_year=&end_mon=&end_year=&ttl_logic=OR&title=&txt_logic=OR&text=&nr_to_return=200&start_nr=1&jou_pick=ALL&ref_stems=&data_and=ALL&group_and=ALL&start_entry_day=&start_entry_mon=&start_entry_year=&end_entry_day=&end_entry_mon=&end_entry_year=&min_score=&sort=SCORE&data_type=SHORT&aut_syn=YES&ttl_syn=YES&txt_syn=YES&aut_wt=1.0&obj_wt=1.0&ttl_wt=0.3&txt_wt=3.0&aut_wgt=YES&obj_wgt=YES&ttl_wgt=YES&txt_wgt=YES&ttl_sco=YES&txt_sco=YES&version=1`}
       />
       <SurveyLink
-        name="TNS"
-        url={`https://wis-tns.weizmann.ac.il/search?&ra=${ra}&decl=${dec}&radius=10&coords_unit=arcsec`}
+        name="CFHT"
+        url={`http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/cadcbin/megapipe/imc.pl?lang=en&object=&size=256&ra=${ra}&dec=${dec}`}
       />
+      <SurveyLink name="CRTS" />
       <SurveyLink
-        name="SNEx"
-        url={`http://secure.lcogt.net/user/supernova/dev/cgi-bin/view_object.cgi?name=iPTF${id}&ra=${ra}&dec=${dec}`}
+        name="DECam"
+        url={`http://legacysurvey.org/viewer?ra=${ra}&dec=${dec}&zoom=14`}
       />
       <SurveyLink
-        name="SIMBAD"
-        url={`http://simbad.u-strasbg.fr/simbad/sim-coo?protocol=html&NbIdent=us=30&Radius.unit=arcsec&CooFrame=FK5&CooEpoch=2000&CooEqui=2000&Coord=${ra}d+${dec}d`}
+        name="DSS"
+        url={`http://archive.stsci.edu/cgi-bin/dss_search?h=5.0&w=5.0&f=fits&v=poss2ukstu_red&r=${ra}d&d=${dec}d&e=J2000&c=none`}
```

Comment:
this tries to download an image, breaking the websocket connection. Try using Stefan's trick for downloads with the finding chart.

Reply:
fixed this with an updated link that displays a gif in a new browser tab instead of downloading a fits file.